query-coordinator/src/main/resources/reference.conf

```diff
@@ -58,9 +58,9 @@ com.socrata.query-coordinator {
   query-timeout = 10m # http client connect timeout
 
   # Previously max-db-query-timeout (default when not specified) was not configurable and hard coded to 12hour.
-  # We are not sure if taking longer than 10 mins after rows start streaming back would be a problem for larger resultsets
+  # We are not sure if taking longer than 60 mins after rows start streaming back would be a problem for larger resultsets
   # If problems arise, we can increase this back to hours in apps_marathon.
-  max-db-query-timeout = 10m
+  max-db-query-timeout = 60m
```

Comment:
in the real config, isn't this overridden by this? https://github.com/socrata/apps-marathon/blob/main/params/fedramp-prod/query-coordinator.toml#L30